src/devices/Media/VideoDevice/samples/Program.cs

```diff
@@ -40,15 +40,19 @@ static async Task Main(string[] args)
             var value = device.GetVideoDeviceValue(VideoDeviceValueType.Rotate);
             Console.WriteLine($"{value.Name} Min: {value.Minimum} Max: {value.Maximum} Step: {value.Step} Default: {value.DefaultValue} Current: {value.CurrentValue}");
 
-            await device.CaptureAsync("/home/pi/jpg_direct_output.jpg");
+            string path = "/home/pi/images";
```

Comment:
why not in current directory? it's usually more intuitive

Reply:
I will submit dockerfile later today

Reply:
I still think current directory would be a better choice, you can remap it to /home/pi/images when running docker container

Reply:
`Directory.GetCurrentDirectory()` does not get the path of the current program. If I want to get the right path, I need to run the `cd` command first, like `cd VideoDevice/sample` `sudo ./VideoDevice.Samples`. When running the docker image, docker does not execute the `cd` command to switch to the program directory. So `Directory.GetCurrentDirectory()` will return "/", and I can't mount the docker's root directory.
If I use the `docker run -w`  parameter to specify the docker directory, the .NET SDK will not be found.